### ISSUE TITLE
Repeated Alias added to apache configuration 

### ISSUE THREAD

**Issue by bigs21** [Neilpang/acme.sh#189]
Everytime that a certificate is issued, the alias to the acme directory is added and the apache backup file is kept. Here's the difference between both files:

> # diff -ub /etc/httpd/conf/httpd.conf /root/.acme.sh/httpd.conf
> --- /etc/httpd/conf/httpd.conf  2016-05-19 19:46:22.006043969 +0100
> +++ /root/.acme.sh/httpd.conf   2016-05-19 19:46:21.943042376 +0100
> @@ -1100,11 +1100,3 @@
>  Allow from all
>  </Directory>
> 
> -
> -Alias /.well-known/acme-challenge  /home/.acme
> -
> -<Directory /home/.acme >
> -Order allow,deny
> -Allow from all
> -</Directory>
> -

I've tested with both an issue of a new certificate without forcing, with the command `acme.sh --issue --apache -d XXXX.tld`, as well as with cron jobs, with the command `/root/.acme.sh/acme.sh --cron --force --home "/root/.acme.sh" > /tmp/acme.sh.log`.

Here's a sample output of the later, having multiple certificates to renew/reissue. Note that in the second group there is one more conflicting alias:
> ...
> [Thu May 19 19:32:56 WEST 2016] The backup file will be deleted on sucess, just forget it.
> [Thu May 19 19:32:56 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1048 will probably never match because it overlaps an earlier Alias.
> Syntax OK
> [Thu May 19 19:32:56 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1048 will probably never match because it overlaps an earlier Alias.
> ...
> [Thu May 19 19:33:16 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1048 will probably never match because it overlaps an earlier Alias.
> [Thu May 19 19:33:16 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1056 will probably never match because it overlaps an earlier Alias.
> Syntax OK
> [Thu May 19 19:33:16 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1048 will probably never match because it overlaps an earlier Alias.
> [Thu May 19 19:33:16 2016] [warn] The Alias directive in /etc/httpd/conf/httpd.conf at line 1056 will probably never match because it overlaps an earlier Alias.
> ...

**Comment by Neilpang** [comment]
I just tried with 2 domains both using `--apache`.

```
acme.sh  --issue  -d  a.com   --apache
acme.sh  --issue  -d  b.com   --apache

acme.sh --renewall  --force

```
But it just works as expected.  no extra alias is there, and the config file is restored .

So, can you please manually fix the config file  and try again?

If it's still not working,  please tell me you OS and apache version.  and  paste the debug log here, with `--debug 2`

https://github.com/Neilpang/acme.sh/wiki/How-to-debug-acme.sh

**Comment by bigs21** [comment]
Here's the version of OS (CentOS Linux) and apache:
```
# cat /etc/redhat-release ; apachectl -V
CentOS release 6.3 (Final)
Server version: Apache/2.2.15 (Unix)
Server built:   Mar 22 2016 19:03:53
Server's Module Magic Number: 20051115:25
Server loaded:  APR 1.3.9, APR-Util 1.3.9
Compiled using: APR 1.3.9, APR-Util 1.3.9
Architecture:   64-bit
Server MPM:     Prefork
  threaded:     no
    forked:     yes (variable process count)
Server compiled with....
 -D APACHE_MPM_DIR="server/mpm/prefork"
 -D APR_HAS_SENDFILE
 -D APR_HAS_MMAP
 -D APR_HAVE_IPV6 (IPv4-mapped addresses enabled)
 -D APR_USE_SYSVSEM_SERIALIZE
 -D APR_USE_PTHREAD_SERIALIZE
 -D SINGLE_LISTEN_UNSERIALIZED_ACCEPT
 -D APR_HAS_OTHER_CHILD
 -D AP_HAVE_RELIABLE_PIPED_LOGS
 -D DYNAMIC_MODULE_LIMIT=128
 -D HTTPD_ROOT="/etc/httpd"
 -D SUEXEC_BIN="/usr/sbin/suexec"
 -D DEFAULT_PIDLOG="run/httpd.pid"
 -D DEFAULT_SCOREBOARD="logs/apache_runtime_status"
 -D DEFAULT_LOCKFILE="logs/accept.lock"
 -D DEFAULT_ERRORLOG="logs/error_log"
 -D AP_TYPES_CONFIG_FILE="conf/mime.types"
 -D SERVER_CONFIG_FILE="conf/httpd.conf"
```

I've deleted the Alias on the end of apache config file, as well as the backup file.
I've also deleted a certificate previously issued on the same machine.

Then, I've issued the certificate with the same parameters (except the domain) that you had, and the issue still persists.
I've posted the log as a gist on https://gist.github.com/bigs21/7c9dc48abc07cf8093c4c9bd82de8e27.

**Comment by Neilpang** [comment]
Hi,
It was a bug, and is fixed at 8f63baf7e4a291324aaee95ecaf0bcbb3df084c7

please try again.
